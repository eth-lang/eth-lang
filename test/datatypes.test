; vim: set ft=clojure:
; line comments
true ; truthy
false
null
123
-123
1.23
-1.23
"asd"
"a\nb"
"a\""
"some
multiline"
""
symbol
name-with-dash
name-with!
bool?
@
@derp
()
(a b c)
[]
[1 2 3]
{}
{a 1 b 2}
{"!" illegal}
