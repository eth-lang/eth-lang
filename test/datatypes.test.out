true;
false;
null;
123;
-123;
1.23;
-1.23;
"asd";
"a\nb";
"a\"";
"some\nmultiline";
"";
symbol;
nameWithDash;
nameWith$;
isBool;
this;
this.derp;
a(b, c);
[1, 2, 3];
{a: 1, b: 2};
{"!": illegal};

